docs/src/main/tut/docs/README.md

```diff
@@ -80,9 +83,11 @@ Abstract definitions it's all it takes to start building programs that support s
 The example below combines both algebras to produce a more complex program
 
 ```tut:book
+import cats.syntax.cartesian._
```

Comment:
IMHO, it is more beginner-friendly to narrow down, as much as possible, what is it that the code snippet is using from this big `cats` library.